kit/ChildSession.cpp

```diff
@@ -1252,7 +1252,7 @@ bool ChildSession::clientZoom(const StringVector& tokens)
 
     if (tokens.size() == 7 &&
         getTokenString(tokens[5], "dpiscale", dpiScale) &&
-        getTokenString(tokens[6], "zoom", zoom))
```

Comment:
This looks like a breaking change (incompatibility core vs online), what could make CI red.

Can we add new property while leaving old one?

So plan would be to:
- merge support in core first for both (zoom and zoompercent in setViewOption)
- then this PR would do the switch

Reply:
ok, it seems CI is green  - nwm

Reply:
Don't think we need to worry the property is not used really yet =)